app/Http/Controllers/PostController.php

```diff
@@ -0,0 +1,101 @@
+<?php
+
+namespace App\Http\Controllers;
+
+use App\Models\Post;
+use Illuminate\Http\Request;
+
+class PostController extends Controller
+{
+    /**
+     * Display a listing of the resource.
+     *
+     * @return \Illuminate\Http\Response
+     */
+    public function index()
+    {
+        return view("insert");
+    }
+
+    /**
+     * Show the form for creating a new resource.
+     *
+     * @return \Illuminate\Http\Response
+     */
+    public function create()
+    {
+        //
+    }
+
+    /**
+     * Store a newly created resource in storage.
+     *
+     * @param  \Illuminate\Http\Request  $request
+     * @return \Illuminate\Http\Response
+     */
+    public function store(Request $request)
+    {
+        $post= new Post;
+        $post->title= $request->get('title');
+        $post->body= $request->get('body');
+        $post->save();
+        return redirect("show");
+    }
+
+    /**
+     * Display the specified resource.
+     *
+     * @param  \App\Models\Post  $post
+     * @return \Illuminate\Http\Response
+     */
+    public function show(Post $post)
+    {
+        $posts = Post::all();
+        return view('show', ['posts'=>$posts]);
```

Comment:
```suggestion
        $posts = Post::all();

        return view('show', compact(['posts']));
```
tương tự nè :3 thêm dòng trắng dãn cách nha. Và dùng compact cho m nhé :3